docs/development/extensions-core/lookups-cached-global.md

```diff
@@ -377,7 +375,8 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 > If using JDBC, you will need to add your database's client JAR files to the extension's directory.
 > For Postgres, the connector JAR is already included.
 > For MySQL, you can get it from https://dev.mysql.com/downloads/connector/j/.
-> Copy or symlink the downloaded file to `extensions/druid-lookups-cached-global` under the distribution root directory.
+> The connector JAR should locate in the classpath of Druid's main class loader.
+> An easy way to add it in the classpath is copying or symlinking the downloaded file to `lib/` under the distribution root directory.
```

Comment:
looks like "symlinking" is called out by spell checker. I think it would be fine to add it to the dictionary in the website module. Might not be in a common dictionary, but from a computing standpoint it is a valid word IMO.

Reply:
I agree it's valid, but deleted it by addressing @techdocsmith's comment below.